settings.gradle

```diff
@@ -184,4 +185,4 @@ include 'data-prepper-plugins:aws-lambda'
 //include 'data-prepper-plugins:dummy-plugin'
 include 'data-prepper-plugin-schema'
 include 'data-prepper-plugins:kinesis-source'
-include 'data-prepper-plugins:opensearch-api-source'
\ No newline at end of file
+include 'data-prepper-plugins:opensearch-api-source'
```

Comment:
Did you mean to add this back?

Reply:
This is a new file. Just creating basic build.gradle so that it can be used in future.